src/memex/views.py

```diff
@@ -82,7 +99,41 @@ def api_config(**settings):
         request_method = ('DELETE', 'GET', 'HEAD', 'POST', 'PUT',)
     settings['request_method'] = request_method + ('OPTIONS',)
 
-    return view_config(**settings)
+    if link_name:
+        link = {'name': link_name,
+                'method': initial_request_method,
```

Comment:
I'm not sure if this works. the `request_method` view config in pyramid can either be a string, or a tuple of strings. One case where it will be a tuple of strings is when we start the move of `PUT /api/annotations/{id}` to `PATCH /api/annotations/{id}`.
Unless our client (and possibly others) know how to deal with a list of request methods?